utils/check.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+# Module that check errors
+module Check
+  def empty?(*_variables)
```

Comment:
Lint/UnderscorePrefixedVariableName: Do not use prefix `_` for a variable that is used.